apps/web/components/ChatPanel/ChatPanel.tsx

```diff
@@ -15,7 +15,7 @@ declare global {
 let boost = null
 
 export const ChatPanel = () => {
-  const [visible, setVisible] = useState(false)
+  const [visible, setVisible] = useState(true)
```

Comment:
I would probably use `react-use` and [`useWindowSize`](https://github.com/streamich/react-use/blob/master/docs/useWindowSize.md) or [`useMedia`](https://github.com/streamich/react-use/blob/master/docs/useMedia.md), so you listen to changes when the window get resized.

You would be able to do then something like:
```tsx
const isBigWindow = useMedia('min-width: 768px')
const [visible, setVisible] = useState(!isBigWindow)
```